core/src/v2/models.rs

```diff
@@ -31,9 +31,10 @@ static PATH_TEMPLATE_REGEX: Lazy<Regex> =
 const SPECIAL_HEADERS: &[&str] = &["content-type", "accept", "authorization"];
 
 /// OpenAPI version.
-#[derive(Clone, Copy, Debug, Serialize, Deserialize, Eq, PartialEq)]
+#[derive(Clone, Copy, Debug, Default, Serialize, Deserialize, Eq, PartialEq)]
```

Comment:
Let's keep default as is for now to avoid bumping msrv?

Reply:
Or maybe it's a good time to bump it, though we'd need to have another breaking release but that's ok I guess